cfgov/jinja2/v1/_includes/blocks/heading.html

```diff
@@ -1,6 +1,41 @@
-<{{ value.level }}>
-    {%- if value.icon %}
-        {{ svg_icon(value.icon) }}
-    {% endif -%}
-    {{ value.text | safe }}
-</{{ value.level }}>
+{# ==========================================================================
+
+   Heading
+
+   ==========================================================================
+
+   Description:
+
+   Create a heading when given:
+
+   value.text:          Heading text.
+
+   value.level:         The heading level (h2, h3, h4). Default: h3.
+
+   value.level_class:   (Optional) Alternate heading class, used for tags like:
+                        <h2 class="h3">
+
+   value.icon:          (Optional) Name of icon to insert before the heading
+                        text. See list of available icon names at:
+                        https://cfpb.github.io/design-system/foundation/iconography
+
+   ========================================================================== #}
+{%- macro heading( value ) -%}
+
+{%- set _default_level = 'h3' -%}
```

Comment:
Rendering the template without `value.level` won't generate a rendering error unless Jinja is configured to use [`StrictUndefined`](https://jinja.palletsprojects.com/en/2.11.x/api/#jinja2.StrictUndefined) (which we don't do for consumerfinance.gov). It'll still render the template but put this message in the HTML:

> {{ no such element: dict object['level'] }}

That said, you make a good point, especially since we seem to set a [default level of h2 when a HeadingBlock is used](https://github.com/cfpb/consumerfinance.gov/blob/ef3597113c1fa6317389841eee5318326518db22/cfgov/v1/blocks.py#L147) in Wagtail.

I've added another commit that removes the default level. I've checked everywhere else this template is used and don't believe there are any other uses where we're not specifying a heading. Would you like to give this another review before I merge?

Reply:
Sounds good. No need to re-review :+1: